### DISCUSSION TITLE
Newbie question: simulation in V and W bands ?

### DISCUSSION THREAD

**Discussion by matt-chv** [thliebig/openEMS-Project#222] [Q&A Usage]
Background:
The example from the tutorial works fine in my setup
[Simple Patch Antenna](https://docs.openems.de/python/openEMS/Tutorials/Simple_Patch_Antenna.html).

I can also successfuly change patch geometry and simulation parameters up to ~10-GHz but then beyond it seems that even when making the simulation resolution << 1/20 of wavelength the simulation does not work (no resonance found)

Question:
Is there an intrinsic limitation to the tool or has someone already used it and have a small example for a patch antenna in either V or W band (i.e. anything between 40 and 110 GHz)?

**Comment by VolkerMuehlhaus** [comment]
There is no intrisic limit, I have used openEMS at >100 GHz. 

The example code might have difficulties to find the dip in S11. It looks for a value < -10dB, but the actual s11 at resonance also depends on the location of the probe feed. So tweak that limit, or just specify the frequency for pattern manually.
